omnibus-base-goerli-andromeda.toml

```diff
@@ -94,3 +97,10 @@ args = [
         { marketId = "<%= imports.perpsFactory.extras.superMarketId %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
     ],
 ]
+
+# Governance Councils Configuration
+[setting.governance_package]
+defaultValue = "synthetix-governance-satellite:3.0.0-rc.0"
+
+[setting.governance_salt]
+defaultValue = "synthetix-council"
```

Comment:
We can use global settings.salt. Any reason yo have another one?

Reply:
We have checks for proxy changes so there cannot be a situation when we unknowingly change salt. If we change salt that means we want to redeploy everything.
Am I missing something more?